test/SafeAnchorSpec.js

```diff
@@ -39,6 +39,17 @@ describe('SafeAnchor', () => {
     handleClick.should.have.been.calledOnce;
   });
 
+  it('provides onClick handler as onKeyDown handler for "space"', () => {
+    const handleClick = sinon.spy();
+
+    tsp(<SafeAnchor onClick={handleClick} />)
+      .shallowRender()
+      .find('a')
+      .trigger('keyDown', { key: ' ', preventDefault() {} });
+
+    handleClick.should.have.been.calledOnce;
```

Comment:
They're equivalent, I was following the pattern of the "it forwards onclick" test https://github.com/andy-j-d/react-bootstrap/blob/b8800dbba51e86ed4bdd34c046f820a67194029c/test/SafeAnchorSpec.js#L39